package.json

```diff
@@ -57,6 +57,7 @@
     "clone": "^1.0.2",
```

Comment:
we already have `clone` - shouldnt need `deep-clone` ( i suspect they do the same thing )